app.getspace.Space.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
```

Comment:
This is something that ideally should be part of your tarball, but it's not a blocker.